process/block/sovereignChainBlock.go

```diff
@@ -1515,15 +1515,28 @@ func (scbp *sovereignChainBlockProcessor) CommitBlock(headerHandler data.HeaderH
 	if err != nil {
 		return err
 	}
+
 	// TODO: MX-15748 Analyse if !check.IfNil(lastMetaBlock) && lastMetaBlock.IsStartOfEpochBlock() from metablock.go
 	err = scbp.commonHeaderAndBodyCommit(headerHandler, body, headerHash, []data.HeaderHandler{lastSelfNotarizedHeader}, [][]byte{lastSelfNotarizedHeaderHash})
 	if err != nil {
 		return err
 	}
 
+	scbp.indexValidatorsRatingIfNeeded(headerHandler)
```

Comment:
I believe this should happen only on epoch start block

Reply:
No, these are the ratings, those are updated at each round. It's the same as it is for meta

Reply:
Actually, you are right.

Reply:
it is only on epochStartBlock.

Reply:
Fixed it